docs/core-concepts/packages.md

```diff
@@ -0,0 +1,170 @@
+
+Packages
+========
+
+.NET Core is split into a set of packages, which provide primitives, higher-level data types, app composition types and common utilities. Each of these packages represent a single assembly of the same name. For example, [System.Runtime](https://www.nuget.org/packages/System.Runtime) contains System.Runtime.dll. 
+
+There are advantages to defining packages in a fine-grained manner:
+
+- Fine-grained packages can ship on their own schedule with relatively limited testing of other packages.
+- Fine-grained packages can provide differing OS and CPU support.
+- Fine-grained packages can have dependencies specific to only one library.
+- Apps are smaller because unreferenced packages don't become part of the app distribution.
+
+Some of these benefits are only used in certain circumstances. For example, NET Core packages will typically ship on the same schedule with the same platform support. In the case of servicing, fixes can be distributed and installed as small single package updates. Due to the narrow scope of change, the validation and time to make a fix available is limited to what is needed for a single library.
+
+The following is a list of the key NuGet packages for .NET Core:
+
+- [System.Runtime](https://www.nuget.org/packages/System.Runtime) - The most fundamental .NET Core package, including [Object](http://dotnet.github.io/api/System.Object), [String](http://dotnet.github.io/api/System.String), [Array](http://dotnet.github.io/api/System.Array), [Action](http://dotnet.github.io/api/System.Action) and [IList&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.IList`1).
+- [System.Collections](https://www.nuget.org/packages/System.Collections) - A set of (primarily) generic collections, including [List&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.List`1) and [Dictionary&lt;K,V&gt;](http://dotnet.github.io/api/System.Collections.Generic.Dictionary`2).
+- [System.Net.Http](https://www.nuget.org/packages/System.Net.Http) - A set of types for HTTP network communication, including [HttpClient](http://dotnet.github.io/api/System.Net.Http.HttpClient) and [HttpResponseMessage](http://dotnet.github.io/api/System.Net.Http.HttpResponseMessage).
+- [System.IO.FileSystem](https://www.nuget.org/packages/System.IO.FileSystem) - A set of types for reading and writing to local or networked disk-based storage, including [File](http://dotnet.github.io/api/System.IO.File) and [Directory](http://dotnet.github.io/api/System.IO.Directory).
+- [System.Linq](https://www.nuget.org/packages/System.Linq) - A set types for querying objects, including Enumerable and [ILookup&lt;TKey, TElement&gt;](http://dotnet.github.io/api/System.Linq.ILookup`2);.
```

Comment:
This is intended to be an inventory, to introduce folks to the packaging. See the wording I've used for the other packages. Does that help?